test/parallel/test-freelist-deprecation.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+// Flags: --expose-internals
+
+const assert = require('assert');
+const freelist = require('freelist');
+const internalFreelist = require('internal/freelist');
+
+assert.equal(typeof freelist, 'object');
+assert.equal(typeof freelist.FreeList, 'function');
+assert.equal(freelist, internalFreelist);
```

Comment:
Sure I ll change it :-), But in this case we established that `freelist` is an Object. So, `strictEqual` would not make any difference here, right?